modules/custom/openy_redirect/openy_redirect.module

```diff
@@ -71,7 +71,7 @@ function openy_redirect_form_node_form_alter(&$form, FormStateInterface $form_st
       if ($i > 0) {
         $form['redirect']['widget'][$i] = $form['redirect']['widget'][$i-1];
       }
-      $form['redirect']['widget'][$i]['value']['#default_value'] = Url::fromUserInput($source['path'], $options);
+      $form['redirect']['widget'][$i]['value']['#default_value'] = Url::fromUserInput('/' . $source['path'], $options)->toString();
```

Comment:
We should avoid double slash.

Reply:
Thank you for this request.
I've fixed this issue.

Reply:
BTW: this won't work if there is an external source in $source['path']